test/createEpicMiddleware-spec.js

```diff
@@ -20,13 +22,29 @@ describe('createEpicMiddleware', () => {
     const mockMiddleware = store => next => action => {
       expect(epic.calledOnce).to.equal(true);
       expect(epic.firstCall.args[0]).to.be.instanceOf(ActionsObservable);
-      expect(epic.firstCall.args[1]).to.equal(store);
+      expect(epic.firstCall.args[1].getState).to.equal(store.getState);
       done();
     };
     const store = createStore(reducer, applyMiddleware(epicMiddleware, mockMiddleware));
     store.dispatch({ type: 'FIRST_ACTION_TO_TRIGGER_MIDDLEWARE' });
   });
 
+  it('should warn about improper use of dispatch function', () => {
+    sinon.spy(console, 'warn');
+    const reducer = (state = [], action) => state.concat(action);
+    const epic = (action$, store) => action$
+      .ofType('PING')
+      ::map(() => store.dispatch({ type: 'PONG' }))
+      ::ignoreElements();
+
+    const middleware = createEpicMiddleware(epic);
+    const store = createStore(reducer, applyMiddleware(middleware));
+
+    store.dispatch({ type: 'PING' });
+
+    expect(console.warn.called).to.equal(true);
```

Comment:
Can you change to `expect(console.warn.callCount).to.equal(1);` or something similar--just want to make sure console.warn is only called once.